scripts/getting-started.sh

```diff
@@ -124,8 +124,9 @@ fi
 
 # Ensure that we have wasm support
 if prompt_default_yes "\n🦀 Setup the Rust environment (e.g. WASM support)?"; then
+    version=`curl -s -H "Accept:application/vnd.github.v3.raw" https://api.github.com/repos/paritytech/polkadot-sdk/contents/rust-toolchain.toml | grep 'channel =' | awk -F'"' '{print $2}'`
     echo "🦀 Setting up Rust environment."
-    rustup default stable
+    rustup default $version
```

Comment:
This command is not necessary at all, it'll use version from `rust-toolchain.toml`. In fact by running this command you revert the effect of `rust-toolchain.toml`, inviting user confusion down the line as the version in `rust-toolchain.toml` keeps changing.

Reply:
I agree, I originally decided to pull the toml file in completely, however Kian raised a point earlier that folks could grab this script and run it from anywhere, not necessarily in their project, but simply as a means to set up their dev env. Suppose that they don't have a `rust-toolchain.toml` in their project, then it would use the latest stable - which we want to avoid.

Reply:
But you're right, since the script potentially brings in the minimal template, maybe the `rust-toolchain.toml` could be added to the minimal template as well?

Reply:
Again, having this line alone defeats the purpose of having `rust-toolchain.toml` completely. They will get an unexpected version of the toolchain and if they are really as unexperienced as this script assumes, they will have no idea why.

Maybe consider copying `rust-toolchain.toml` if it does not yet exists or something, though except for contributions to polkadot-sdk they don't have to use the same exact version, any current or newer stable toolchain will work just fine, hence suggestion to remove this completely.

Reply:
Thanks for all the input, I've pushed some changes to accommodate that. It will pull the toml directly into the minimal template folder that gets generated by the script.